front/src/routes/map/Map.jsx

```diff
@@ -4,6 +4,7 @@ import 'leaflet/dist/leaflet.css';
 import style from './style.css';
 
 const DEFAULT_COORDS = [48.8583, 2.2945];
+var markerArray = [];
```

Comment:
Warning, here you're creating a global variable ! ⚠️ 

DEFAULT_COORDS is global because it's a const that will never change.

Any variable which is modified during the lifecycle of an object should be in the component.

Reply:
More info 

https://www.freecodecamp.org/news/where-do-i-belong-a-guide-to-saving-react-component-data-in-state-store-static-and-this-c49b335e2a00/